README.md

```diff
@@ -10,3 +10,156 @@ https://appliedzkp.github.io/maci/
 
 We welcome contributions to this project. Please join our
 [Telegram group](https://t.me/joinchat/LUgOpE7J2gstRcZqdERyvw) to discuss.
+
+## Local Development and testing
+
+### Requirements
+
+You should have Node 14 installed. Use `nvm` to install it and manage versions.
+
+You also need a Ubuntu/Debian Linux machine on an Intel CPU.
+
+### Get started
+
+Install dependencies:
+`sudo apt-get install build-essential libgmp-dev libssl-dev libsodium-dev git nlohmann-json3-dev nasm g++ libgcc-s1`
+
+If you are missing the correct version of glibc see `circuits/scripts/installGlibc.sh`
+
+Clone this repository, install NodeJS dependencies, and build the source code:
+
+```bash
+git clone git@github.com:appliedzkp/maci.git && \
+cd maci && \
+npm i && \
+npm run bootstrap && \
+npm run build
+```
+
+For development purposes, you can generate the proving and verifying keys for
+the zk-SNARK circuits, along with their Solidity verifier contracts as such.
+
+Install Rust:
```

Comment:
For v1, we should add instructions for rapidsnark instead of zkutil.

Reply:
Ah yes good catch, do you know of a way to preserve the contribution history as well?